JavaComparator/Comparator.java

```diff
@@ -0,0 +1,133 @@
+import java.util.ArrayList;
+
+public class Comparator {
+    String file_orig, file_tripped;
+    String file_extension;
+
+    Comparator(String Path1, String Path2) {
+        file_orig = Path1;
+        file_tripped = Path2;
+
+        if (Path1.endsWith("docx") && Path2.endsWith("docx")) {
+            file_extension = "docx";
+        } else if (Path1.endsWith("pptx") && Path2.endsWith("pptx")) {
+            file_extension = "pptx";
+        } else if (Path1.endsWith("xlsx") && Path2.endsWith("xlsx")) {
+            file_extension = "xlsx";
+        } else {
+            file_extension = "invalid";
+            System.out.println("Either Path is invalid / File types do not match");
```

Comment:
Change the Print statements to be added to DEBUG.txt file.

Reply:
+1